build/openwisp_base/Dockerfile

```diff
@@ -4,7 +4,7 @@ FROM python:3.7.5-slim-buster AS SYSTEM
 # 1. Pillow: zlib1g-dev libjpeg-dev
 # 2. gettext: Required by envsubst used in scripts
 RUN apt update && \
-    apt install --yes --no-install-recommends zlib1g-dev libjpeg-dev gdal-bin gettext && \
+    apt install --yes --no-install-recommends zlib1g-dev libjpeg-dev gdal-bin gettext libcairo2 libpango-1.0-0 libpangocairo-1.0-0 && \
```

Comment:
Also, change this as followed:

```
# System requirements:
# 1. Pillow: zlib1g-dev libjpeg-dev
# 2. gettext: Required by envsubst used in scripts
# 3. django-freeradius/weasyprint: libcairo2 libpangocairo-1.0
RUN apt update && \
    apt install --yes --no-install-recommends zlib1g-dev libjpeg-dev \
                libcairo2 libpangocairo-1.0 gdal-bin gettext && \
    rm -rf /var/lib/apt/lists/* /root/.cache/pip/* /tmp/*
```

Reply:
I made https://github.com/openwisp/docker-openwisp/pull/79 because I had to fix another issue which was stopping another GCI task, I'll merge it after @2stacks reviews it and you can rebase to it! :smile:

Reply:
I stilk have to change this right?

Reply:
Rebase to master! :smile: